Tools/MonoGame.Content.Builder.Editor/Common/PipelineController.cs

```diff
@@ -105,11 +105,22 @@ private PipelineController(IView view)
 
             _templateItems = new List<ContentItemTemplate>();
             var root = Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location);
-            if (Directory.Exists(Path.Combine (root, "..", "Resources", "Templates")))
+            var macPath = Path.Combine(root, "..", "Resources");
+            var windowsAndLinuxPath = Path.Combine(root, "Templates");
+
+            if (Directory.Exists(macPath))
+            {
+                LoadTemplates(macPath);
+            }
+            else if (Directory.Exists(windowsAndLinuxPath))
             {
-                root = Path.Combine(root, "..", "Resources");
+                LoadTemplates(windowsAndLinuxPath);
             }
-            LoadTemplates(Path.Combine(root, "../../../Templates"));
+            else
+            {
+                throw new DirectoryNotFoundException("'Templates' or 'Resources' folder not found");
```

Comment:
We don't want to crash the application if the file templates are missing XD

Reply:
Ok, fair enough. Will remove it.